### ISSUE TITLE
BIP39 'custom words' should be 'passphrase'

### ISSUE THREAD

**Issue by markblundeberg** [Electron-Cash/Electron-Cash#958]
I was trying to recover my Bitcoin.com wallet seed today and banging my head against the wall with getting completely empty wallets of entirely different addresses, until I realized I had a [BIP39 passphrase](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#from-mnemonic-to-seed) set up.

In the Bitcoin.com seed display page, it said "This recovery phrase was created with a password. To recover this wallet both the recovery phrase and password are needed."... eventually I figured it out: I should check the box that says 'extend with custom words' and enter the passphrase that I had originally put into the 'Advanced Options' of the bitcoin.com wallet creation.

![screenshot from 2018-11-18 22-49-43](https://user-images.githubusercontent.com/36528214/48692044-863d2080-eb89-11e8-96d1-3c3b25bbd51f.png)

So, basically, "extend with custom words" thing is a misnomer in case of BIP39.

**Comment by kyuupichan** [comment]
That is deliberate and from about 2 years ago in Electrum.   If you change it put (a.k.a. "passphrase") in quotes and brackets.  Keep the wording though.  People confuse passphrase with password; it was a mistake of BIP39 to use that.

**Comment by cculianu** [comment]
FYI re: translations.

If you do keep the old wording + add new wording in parens note the translations would break.  

You can either:

1. Not care about translations breaking (perfectly ok -- we will get new translations eventually). And do 

eg `_("Extend this seed with custom words (aka 'passphrase')")`

2. Care about translations breaking, and try and preserve the existing one:

eg `_("Extend this seed with custom words") + _(" (aka 'passphrase')")`

Or whatever else you feel like. Your call!
